extensions/import/src/common/constants.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+export const extensionConfigSectionName = 'flatFileImport';
+export const serviceName = 'Flat File Import Service';
+export const providerId = 'FlatFileImport';
+export const configLogDebugInfo = 'logDebugInfo';
+export const sqlConfigSectionName = 'sql';
+export const mssqlProvider = 'MSSQL';
+
+export const supportedProviders = [mssqlProvider];
+
+// Links
+export const serviceCrashLink = 'https://github.com/Microsoft/azuredatastudio/issues/2090';
```

Comment:
Should this be a aka.ms link?

@Charles-Gagnon

Reply:
Doesn't really matter to me - it's unlikely to ever change so I think it's fine as it is.